### PR TITLE
Feature 2.0 streaming

### PR DESCRIPTION
**Pull request by fredrik-talkamatic**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,66 +1,23 @@
...
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
+  "author": "Talkamatic AB",
+  "packageManager": "yarn@4.0.2",
+  "dependencies": {
+    "speechstate": "https://git@github.com/vladmaraev/speechstate.git",
```
Comment (vladmaraev) [inline package.json:14]: I am still not sure that this is a viable practice, given that the dependency will be locked (`yarn.lock`). Also, potentially, there can be newer releases of speechstate that might break compatibility with tala-speech. I would prefer having a specific version of speechstate here.

File: package.json [hunk 2]
```diff
@@ -1,66 +1,23 @@
...
+    "typescript": "^5.3.3",
+    "typescript-language-server": "^4.3.1",
+    "vite": "^5.0.11",
+    "xstate": "^5.9.1"
```
Comment (vladmaraev) [inline package.json:18]: Just a note. Ideally, this should be using the version of xstate that speechstate is using.

**Review by vladmaraev — COMMENTED** [review]
See my comments below. Additionally, we need to point the instances that are using tala-speech-1.0 to use either v1.1.0 (without streaming) or, first, release the version v.1.2.0 with streaming support (current master) and then point them to this version.

File: package.json (hunk 1 above)
Comment (fredrik-talkamatic) [reply package.json:14]: Then we need to solve this in another way. I'm very unskilled in the craft of releases and distribution of JS packages, but let's select the simplest solution and do it.

File: package.json (hunk 2 above)
Comment (fredrik-talkamatic) [reply package.json:18]: So should we downdate this to 5.5.1, or update speechstate to 5.9.1? 

I guess that updating ss to 5.9.1 would mean releasing 2.0.1 of ss.

**Comment by fredrik-talkamatic** [comment]
I updated the package.json so that speechstate points to 2.0.0.

File: package.json (hunk 1 above)
Comment (fredrik-talkamatic) [reply package.json]: I changed the dependency to 2.0.0. Should work now.

Comment (fredrik-talkamatic) [reply package.json]: BTW, Vlad - does the yarn.lock file need to be in the repo, since it's a generated file?

File: package.json (hunk 2 above)
Comment (vladmaraev) [reply package.json:18]: The latter. Didn't we release 2.0.0 of speechstate? I can't see it now.

File: package.json (hunk 1 above)
Comment (vladmaraev) [reply package.json]: Great, you can see that it builds now!

**Pull request merged**